config/300-bus.yaml

```diff
@@ -27,3 +26,6 @@ spec:
     - all
     - knative
     - eventing
+    shortNames:
+    - kbus
+  scope: Namespaced
```

Comment:
Why move this to the end?

Reply:
to be constant with the rest of the resource definitions in the project